### DISCUSSION TITLE
feat(capacitor-live-update): Add differential updates

### DISCUSSION THREAD

**Discussion by Eraldo** [capawesome-team/capacitor-plugins#178] [Ideas]
Concept: https://en.wikipedia.org/wiki/Delta_update

Ratio: To increase speed and decrease traffic. 🎉 

Happy to talk about different implementation strategies and use cases. 👍

**Comment by robingenz** [comment]
I created a feature request for this: https://github.com/capawesome-team/capacitor-plugins/issues/180
